sqlalchemy_ibmi/base.py

```diff
@@ -43,6 +45,11 @@
 * ``readonly`` - If ``True``, the connection is set to read-only. Defaults to ``False``.
 * ``timeout`` - The login timeout for the connection, in seconds.
 
+create-engine arguments:
+
+* ``fast_executemany`` - Specify the pyodbc `fast_executemany <https://github.com/mkleehammer/pyodbc/wiki/Cursor#executemanysql-params-with-fast_executemanytrue>`_ option.
+Defaults to ``False``. Inserting a Decimal object into a Float column is not supported with this functionality.
```

Comment:
I would suggest being more generic here, but still give the concrete known example. eg.

Little to no conversion is supported between input and target types when this flag is enabled. Ensure the types of the supplied parameters match or errors can occur, eg. inserting a Decimal object in to a FLOAT column will produce the error "Converting decimal loses precision".

Reply:
This should also be moved before the "Defaults to False` line.